src/test/clojurescript/clara/rule_defs.cljs

```diff
@@ -0,0 +1,42 @@
+(ns clara.rule-defs
+  (:require-macros [cljs.test :refer (is deftest run-tests testing)]
+                   [clara.test-rules-data])
+  (:require [cljs.test :as t]
+            [clara.rules.engine :as eng]
```

Comment:
I don't see any direct usage of the clara.rules.engine in this namespace, so I think this require can be removed.  I wouldn't expect such usage since that namespace isn't intended to be a public API.